traveler-swift-core/TravelerKit/Models/Flight.swift

```diff
@@ -30,6 +30,10 @@ public struct Flight: Codable, Equatable {
     public let departureDate: Date
     /// Arrival time
     public let arrivalDate: Date
+    /// Departure terminal
+    public let departureTerminal: String?
+    /// Arrival terminal
+    public let arrivalTerminal: String?
```

Comment:
The terminals are Airport information not flight information, these fields belong in that model.

Reply:
These are being returned in the Flight model currently, need to have backend change this to move to the Airport model

Reply:
Added TODO